### ISSUE TITLE
häufiger TIMEOUT-Fehler - CMD_CAMERA_INFO und CMD_SET_ARMING

### ISSUE THREAD

**Issue by mchlmng** [bropat/ioBroker.eufy-security#151]
Hallo und vielen Dank erstmal für die Bereitstellung des Adapters.

Im laufenden Betrieb tauchen sehr häufig Fehler auf. Es scheint, dass sie aus der regelmäßigen Aktualisierung resultieren. Der Abstand der Fehler entspricht dem Intervall aus den Instanz-Einstellungen. Der Modus wird in meiner Umgebung in einem Node-Red-Flow geprüft und je nach Trigger gesetzt.

```
2021-07-25 20:11:47.005
error
Station: T8010Nxxx... command CMD_CAMERA_INFO failed with error: ERROR_COMMAND_TIMEOUT (-133)

2021-07-25 20:11:47.005
warn
Station T8010Nxxx... - Result data for command not received [{"message": {"sequence":46,"command_type":1103,"channel":255,"data":{"type":"Buffer","data":[209,0,0,46,88,90,89,72,79,4,4,0,0,0,1,0,255,0,0,0,255,0,0,0]},"retries":1,"acknowledged":true,"return_code":-133,"timeout":{"_idleTimeout":20000,"_idlePrev":null,"_idleNext":null,"_idleStart":21633768,"_repeat":null,"_destroyed":false}}}]

2021-07-25 20:05:20.448
error
Station: T8010Nxxx... command CMD_SET_ARMING failed with error: ERROR_COMMAND_TIMEOUT (-133)

2021-07-25 20:05:20.448  **
![2021-07-25 20_21_47-logs - iobroker](https://user-images.githubusercontent.com/87946308/126909316-28bbdf63-4fc7-44a4-8879-89115ef878ad.png)
Dieses Ereignis resultiert aus einem Node-Red-Flow, der den Status ändern soll**
warn
Station T8010Nxxx... - Result data for command not received [{"message":{"sequence":45,"command_type":1350,"nested_command_type":1224,"channel":255,"data":{"type":"Buffer","data":[209,0,0,45,88,90,89,72,70,5,134,0,0,0,1,0,255,0,0,0,123,34,97,99,99,111,117,110,116,95,105,100,34,58,34,57,48,99,102,52,100,97,101,99,53,48,57,56,100,53,97,49,57,53,51,48,50,99,97,53,57,97,102,56,50,99,54,48,100,99,54,56,97,99,102,34,44,34,99,109,100,34,58,49,50,50,52,44,34,109,86,97,108,117,101,51,34,58,48,44,34,112,97,121,108,111,97,100,34,58,123,34,109,111,100,101,95,116,121,112,101,34,58,49,44,34,117,115,101,114,95,110,97,109,101,34,58,34,109,99,104,108,109,110,103,46,106,114,115,98,107,34,125,125]},"retries":1,"acknowledged":true,"return_code":-133,"timeout":{"_idleTimeout":20000,"_idlePrev":null,"_idleNext":null,"_idleStart":21247212,"_repeat":null,"_destroyed":false}}}]
```
![2021-07-25 20_21_47-logs - iobroker](https://user-images.githubusercontent.com/87946308/126909324-c9ac888b-a082-4b6e-9d95-0b9e5e263ee0.jpg)

Kann ich etwas verändern, um den Fehler zu vermeiden?

Vielen Dank!

**Comment by Stormbringer11** [comment]
Habe auch durchgehend ähnlichen Fehler im LOG:

eufy-security.0 | 2021-07-28 13:17:17.355 | error | (1168408) Station: TXX command CMD_CAMERA_INFO failed with error: ERROR_COMMAND_TIMEOUT (-133)
-- | -- | -- | --
eufy-security.0 | 2021-07-28 13:17:17.355 | warn | (1168408) Station TXX - Result data for command not received [{"message":{"sequence":472,"command_type":1103,"channel":255,"data":{"type":"Buffer","data":[209,0,1,216,88,90,89,72,79,4,4,

**Comment by bropat** [comment]
@Stormbringer11 @mchlmng 

Der Rest funktioniert einwandfrei?
Habt ihr bereits probiert die Eufy Station neuzustarten?

**Comment by Stormbringer11** [comment]
ja perfekt, bis auf das senden von Bildern per Telegram (da wird immer das gleiche alte Bild gesendet) aber denke das wird ein anderes Problem sein

**Comment by Stormbringer11** [comment]
Neustart hilft nichts

**Comment by bropat** [comment]
@Stormbringer11 

Welche Geräte sind an der Station registriert? (Txxxx mehr brauch ich nicht)

**Comment by Stormbringer11** [comment]
2 Eufy Cam mit T8114 und T8140 beginnend und eine Klingel mit T8210

**Comment by bropat** [comment]
@Stormbringer11 

ok dann bitte ich dich den Adapter in [Debug-Modus zu versetzen](https://github.com/bropat/ioBroker.eufy-security/wiki/Howto-enable-debug), den Fehler 1 Mal nachzubilden und mir dann die Logs zuzusenden (meine Mail findest du [hier](https://github.com/bropat/ioBroker.eufy-security#license)).

**Comment by Stormbringer11** [comment]
Hast Post. Hoffe es hilft dir weiter ansonsten einfach nochmal schreien 😉 Vielen Dank im Voraus

**Comment by bropat** [comment]
Ich habe nur eine leere Mail erhalten ohne Anhang.

**Comment by Stormbringer11** [comment]
Komisch, ich seh alles unter gesendet. Sorry, siehst jetzt was?

**Comment by bropat** [comment]
@Stormbringer11 Probiere die letzte Version des Adapters 0.6.1.

**Comment by Stormbringer11** [comment]
Danke aber der Fehler kommt immernoch  rein:

eufy-security.0	2021-08-19 19:25:15.175	error	(547618) Station: T8010P2320280268 command CMD_CAMERA_INFO failed with error: ERROR_COMMAND_TIMEOUT (-133)
eufy-security.0	2021-08-19 19:25:15.175	debug	(547618) [Station.onCommandResponse] Got p2p command response [{"station":"T8010P2320280268","commandType":1103,"channel":255,"returnCodeName":"ERROR_COMMAND_TIMEOUT","returnCode":-133}]
eufy-security.0	2021-08-19 19:25:15.175	warn	(547618) Station T8010P2320280268 - Result data for command not received [{"message":{"sequence":6,"command_type":1103,"channel":255,"data":{"type":"Buffer","data":[209,0,0,6,88,90,89,72,79,4,4,0,0,0


Der neue  DP eufy-security.0.station.switch_mode_with_access_code, wäre cool aber der ist irgendwie ohne Funkion. Wenn man über Keypad öffnet bleibt er auf false, falls dich das interessiert ;o)

**Comment by bropat** [comment]
> Danke aber der Fehler kommt immernoch  rein:

Dache ich mir, hatte hierfür noch nichts verändert :P

> Der neue  DP eufy-security.0.station.switch_mode_with_access_code, wäre cool aber der ist irgendwie ohne Funkion. Wenn man über Keypad öffnet bleibt er auf false, falls dich das interessiert ;o)

Das ist eine Einstellungsmöglichkeit (findest du natürlich auch in der App), kein Status ;)

**Comment by DivanX10** [comment]
I don't know how it turned out, but I tried different ibroker images in docker and after installing the buanet/iobroker:v5.1.0-amd64 image, everything worked fine.

**Comment by DivanX10** [comment]
Что-то ужасное творится с этой интеграцией. Сильно грузит iobroker. В сообщении сообщается

```
Problems with frequent failures of adapter instances
The adapter instance often fails at startup and was stopped because of this. Before restarting the instance, you need to check the log file.
```
![image](https://user-images.githubusercontent.com/64090632/131891987-d31b90da-b5f1-48c3-96f5-14a14fdb22d8.png)
![image](https://user-images.githubusercontent.com/64090632/131891844-2e859aa3-5a3f-4964-9645-45cc4c46dfb0.png)
![image](https://user-images.githubusercontent.com/64090632/131891945-6caff306-beba-42ab-af62-1b8eeabc85d3.png)

**Comment by bropat** [comment]
@DivanX10 
Are you still having timeout problems?

**Comment by DivanX10** [comment]
> @DivanX10
> Are you still having timeout problems?

Yes, the problem persisted and I turned off the neobroker, because the camera is going crazy. Then it turns itself in all directions, then just the neobroker loads the system heavily and thanks to the limiter, the container is being reloaded. The neobroker is now turned off.

**Comment by mchlmng** [comment]
I don't have any problems since I run the adapter with log level "debug". Most of the time a single error only appears directly after calling the iobroker admin-page.

**Comment by DivanX10** [comment]
I decided to install IOBROKER_V5.2.0-BETA4-AMD64, it seems that the situation has become better. With IOBROKER 5.1.0, everything is sad, there is a direct problem with this driver for the easy camera. Errors are poured in such a way that IOBROKER forcibly turns off the driver for eufi integration.

![image](https://user-images.githubusercontent.com/64090632/134886764-5355ac4e-7721-4bff-aa08-18a2ef0c7917.png)
![image](https://user-images.githubusercontent.com/64090632/134886937-5c33063d-824b-4ee3-91ae-9bd7ae528234.png)

**Comment by SmartM-ui** [comment]
Hi @bropat ,
Thanks to the help of @DivanX10 
i installed io-brocker and i noticed the same problem which is reported every 10 minutes:

2021-09-30 15:22:53.690 error Station: T8410P20210902AA command CMD_CAMERA_INFO failed with error: ERROR_COMMAND_TIMEOUT (-133)

Hope you will solve it soon

Thanks
